stable/mattermost-team-edition/templates/pvc.yaml

```diff
@@ -0,0 +1,28 @@
+{{ if and .Values.persistence.data.enabled (not .Values.persistence.data.existingClaim) }}
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ template "mattermost-team-edition.fullname" . }}
+  labels:
+    chart: {{ template "mattermost-team-edition.chart" . }}
+    app: {{ template "mattermost-team-edition.fullname" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  annotations:
+  {{ range $key, $value := .Values.persistence.data.annotations }}
+    {{ $key }}: {{ $value | quote }}
+  {{ end }}
+spec:
```

Comment:
Add `selector.matchLabels` including `app` and `release` labels.

Reply:
@unguiculus If enable the matchLabels for the PVC the PVC is in pending forever. I guess then we need to create the PV and add the labels there to work

![screen shot 2018-09-17 at 09 35 45](https://user-images.githubusercontent.com/4115580/45610676-157c6980-ba5d-11e8-9136-efe09700e94b.png)

Reply:
I was wrong. Dynamic provision is not possible if `matchLabels` is set.